lib/we/call/connection.rb

```diff
@@ -110,14 +110,14 @@ def adapter_handles_gzip?(adapter)
       end
 
       def setup_sunset_middleware(faraday)
-        # In v0.5.0 this was a bool switch, now it takes :active_support or an instance of a Logger
-        if config.detect_deprecations == true || config.detect_deprecations == :active_support
-          return { active_support: true }
-
+        options = { rollbar: :auto }
```

Comment:
if this method isn't opt-in, this could make a system very noisy and actually trigger pager duty incidents depending on the team's setup.

Reply:
this would throw a rollbar warning (not error) only in the event of a sunset header being detected with rollbar configured - I don't think it would make things too noisy

Reply:
Yeah this is designed to silently fail if there is no rollbar, and use it if there is rollbar. I think we’ll be ok.

Reply:
@bagmangood if an application is using deprecated endpoints they should get warnings, because things are about to break. You'ere describing the intended purpose of this change like an unexpected side effect. We are intentionally bubbling deprecating warnings up to people, and we want them to notice them. Subtly sneaking them into logs that nobody is looking at is not helping anyone.

Reply:
I don't disagree that they _should_ be warnings (or that logs are certainly not effective). I'm concerned about the noisiness of the warnings (as well as the fact that rollbar pricing is based on number of alerts)

Reply:
Totally understand, but it seems like all incentivization is pointing to fixing the problem as soon as possible, which is what we want.